src/main/java/frc/robot/Robot.java

```diff
@@ -16,8 +18,19 @@
 public class Robot extends TimedRobot {
 	private TeleopInput input;
 
+	//Predefined auto paths
```

Comment:
I wonder if we could put these path definitions inside the auto handler class. Maybe AutoHandlerSystem exposes a public AutoFSMPathId enum and keeps the corresponding paths hidden within the class. That way we can avoid having to touch Robot.java any time the path needs to change.